src/modules/tasks.js

```diff
@@ -0,0 +1,211 @@
+// eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files